deploy.yml

```diff
@@ -104,8 +130,14 @@ Resources:
               },
               "build": {
                 "commands": [
+                  "echo 'Build phase...'",
                   "git clone https://github.com/aws-samples/bedrock-claude-chat.git",
-                  "cd bedrock-claude-chat/cdk",
+                  "cd bedrock-claude-chat",
+                  "if [ \"$ALLOW_SELF_REGISTER\" = \"false\" ]; then sed -i 's/selfSignUpEnabled: !props.idp.isExist()/selfSignUpEnabled: false/' cdk/lib/constructs/auth.ts; fi",
```

Comment:
Could you please make the `selfSignUpEnabled` configurable on `cdk.json` as well as ipv4, ipv6 because `auth.ts` might be changed on future updates. Also modifying auth.ts like following would be helpful.
```ts
selfSignUpEnabled: props.selfSignUpEnabled && !props.idp.isExist(),
```

Reply:
Updated